### DISCUSSION TITLE
Multiple Widgets causes visitor to register for each widget

### DISCUSSION THREAD

**Discussion by netz0** [LiveHelperChat/livehelperchat#1975] [General]
In the docs, it says that multiple widgets are supported. 

Reference:
[https://doc.livehelperchat.com/docs/javascript-arguments#how-to-embed-multiple-widgets-on-the-same-page](https://doc.livehelperchat.com/docs/javascript-arguments#how-to-embed-multiple-widgets-on-the-same-page)

And it even explains a case scenario that would normally require this:

_E.g you have a page embed code and you want to have a widget always on bottom-left corner._

This works, you can now have a widget and an embed on the same page, a combination as long as they use a different ID.

But...

On the admin side, on the visitor lists, each new visitor will show now twice, once for each widget. This must be a new bug as older releases tracked the visitors only once, regardless if there were multiple widgets on the same page.

**Comment by remdex** [comment]
I don't know which version you refer, but it was always like that... As each embed is entirely independent of the other...
https://livehelperchat.com/demo-12c.html

If you want to extend you are always free to do so and create a pull request.

**Reply by netz0** [reply]
I see, maybe then adding a filter on the view admin side could do the trick to avoid showing every visitor twice. I will create a request.
